packages/next/build/output/index.ts

```diff
@@ -100,6 +107,8 @@ export function formatAmpMessages(amp: AmpPageStatus) {
     messages.push(['', '', '', ''])
   }
 
+  if (!messages.length) return []
```

Comment:
Return type is a `string` afaik:
```suggestion
  if (!messages.length) {
    return ''
  }
```